### PR TITLE
Fix up fine tooth comb issues. 

### PR DESCRIPTION
**Pull request by felixs8696**
Pages reviewed:
- Introduction
- Getting Started
- Model Zoo
- Guides/Completions

Checklist:
- [x] Run all code snippets
- [x] Try all alternate paths pointed to in the docs
- [x] Fix any language issues or typos
- [x] Click every link and ensure not broken
- [x] Generally review with extreme detail and make a PR to fix any issues

### CODE REVIEW

File: docs/getting_started.md [hunk 1]
```diff
@@ -1,17 +1,16 @@
 # Getting Started
 
-The fastest way to get started with LLM Engine is to use the python client in this repository to run inference and fine-tuning on Scale's infrastructure. This path does not require you to install anything on your infrastructure, and Scale's free tier gives you access to experimentation using open source LLMs.
+The fastest way to get started with LLM Engine is to use the Python client in this repository to 
+run inference and fine-tuning on Scale's infrastructure. This path does not require you to install 
+anything on your infrastructure, and Scale's free tier gives you access to experimentation using 
+open source LLMs.
 
-To start with, install LLM Engine via pip or conda:
+To start with, install LLM Engine via pip:
```
Comment (yixu34) [inline docs/getting_started.md:8]: `To start,`?

**Review by rkaplan — APPROVED** [review]

File: docs/guides/completions.md [hunk 2]
```diff
@@ -15,12 +20,19 @@ response = Completion.create(
...
+
+print(response.output.text)
+# ________ and I am a ________
 ```
 
-The _model_name_ is the LLM to be used (see [Model Zoo](../../model_zoo)).
-The _prompt_ is the main input for the LLM to respond to.
-The _max_new_tokens_ parameter is the maximum number of tokens to generate in the chat completion.
-The _temperature_ is the sampling temperature to use. Higher values make the output more random, while lower values will make it more focussed and deterministic.
+**model:** The LLM you want to use (see [Model Zoo](../../model_zoo)).
```
Comment (yixu34) [inline docs/guides/completions.md:31]: Should we make these bullets?

File: docs/guides/completions.md [hunk 3]
```diff
@@ -41,34 +58,41 @@ An example Completion API response looks as follows:
...
 stream = Completion.create(
-    model="llama-7b",
-    prompt="why is the sky blue?",
-    max_new_tokens=5,
+    model="falcon-7b-instruct",
+    prompt="Give me a 200 word summary on the current economic events in the US.",
+    max_new_tokens=1000,
     temperature=0.2,
-    stream=True,
+    stream=True
```
Comment (yixu34) [inline docs/guides/completions.md:89]: nit: you can leave the comma if you want. I think it's idiomatic to leave the trailing commas in python

File: docs/guides/completions.md [hunk 4]
```diff
@@ -41,34 +58,41 @@ An example Completion API response looks as follows:
...
 )
 
 for response in stream:
     if response.output:
-        print(response.json())
+        print(response.output.text, end="")
+        sys.stdout.flush()
```
Comment (yixu34) [inline docs/guides/completions.md:95]: nice

File: docs/index.md [hunk 5]
```diff
@@ -31,11 +30,11 @@ Kubernetes.
 ### Key Features
 
 **Ready-to-use APIs for your favorite models**: Deploy and serve
-open-source foundation models - including LLaMA, MPT and Falcon.
+open-source foundation models - including LLaMA, MPT, and Falcon.
```
Comment (yixu34) [inline docs/index.md:33]: Hmm I think we want `open source`?

File: docs/guides/completions.md (hunk 2 above)
Comment (felixs8696) [reply docs/guides/completions.md:31]: Which do you prefer?

Screenshot 2023-07-17 at 11 01 10 PM
Screenshot 2023-07-17 at 11 01 31 PM

Comment (felixs8696) [reply docs/guides/completions.md:31]: I though the bullets added unnecessary spacing

File: docs/guides/completions.md (hunk 3 above)
Comment (felixs8696) [reply docs/guides/completions.md:89]: Ah yes, copied this from the getting started page. Put it back

File: docs/guides/completions.md (hunk 2 above)
Comment (yixu34) [reply docs/guides/completions.md:31]: Oooh yeah you're right

**Review by yixu34 — APPROVED** [review]

**Pull request merged**